packages/name-controller/src/NameController.ts

```diff
@@ -14,6 +14,17 @@ import type {
 import { NameType } from './types';
 
 export const FALLBACK_VARIATION = '*';
+
+/**
+ * Enumerates the possible origins responsible for setting a petname.
+ */
+export enum ORIGIN {
```

Comment:
Re don't capitalize enum name: Done

I vote that we keep the Origin enum in this file because it's only used with SetNameRequest or NameEntry, which are both defined here. Although moving it to types.ts might allow for tree-shaking, this isn't necessary since Origin only gets used when this API is invoked. Additionally, maintaining it in the same file provides a clear context and readability.

Reply:
Done.